webpack/JobInvocationDetail/JobInvocationHostTable.js

```diff
@@ -81,6 +99,34 @@ const JobInvocationHostTable = ({ id, targeting, finished, autoRefresh }) => {
     `/${controller}/auto_complete_search`
   );
 
+  const constructFilter = () => {
+    const baseFilter = `job_invocation.id = ${id}`;
+    const dropdownFilterClause =
+      selectedFilter && selectedFilter !== 'all_statuses'
+        ? `and job_invocation.result = ${selectedFilter}`
+        : '';
```

Comment:
@adamruzicka Is it correct to filter `/api/job_invocations/53/hosts?search=` with  `job_invocation.id = 53 and job_invocation.result = success and (name ~  centos9 )` ? It does provide the correct results

Reply:
It is a bit counter-intuitive, but this indeed seems to be the right way. I was thinking about dropping the `job_invocation.id = 53`, but looking at the sql that would generate we would get hosts which were in this job, but succeeded in another.